### PR TITLE
Setup folder structure and contribution rules

### PR DESCRIPTION
**Pull request by LiyouZhou**
README.md describe the folder structure and defines for format for model
contribution. Added initial folders for camera and lidar perception.
Added Apache-2.0 LICENSE.

Change-Id: Icd61a4262e05c371babc623a2511a4308d110125
Signed-off-by: Liyou Zhou <liyou.zhou@arm.com>
Issue-Id: SCM-875

### CODE REVIEW

**Comment by LiyouZhou** [comment]
@gbiggs @JWhitleyWork @servandoGerman This patch sets up the basic structure of the model zoo. would love any feedback.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,164 @@
...
+model files can be specified using the license field in
+[definition.yaml](#filling-out-the-definition-file).
+
+## Cloning The Model Zoo
+
+You need to [install Git LFS](https://git-lfs.github.com/) for large file
+support. To clone the zoo:
+
+```sh
+$ GIT_LFS_SKIP_SMUDGE=1 git clone git@github.com:autowarefoundation/modelzoo.git
```
Comment (sgermanserrano) [inline README.md:20]: @LiyouZhou shouldn't this be https?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,164 @@
...
+A collection of machine-learned models for use in autonomous driving
+applications.
+
+## License
+
+All source code published in this repository by default are licensed under
+Apache 2.0. If contributions need to be made under a difference license, the
+LICENSE body need to be included in the sub-folder of the model. License for the
+model files can be specified using the license field in
+[definition.yaml](#filling-out-the-definition-file).
```
Comment (JWhitleyWork) [inline README.md:12]: It's very important that the license for the model be specified separately from the license for the training data. This is a major issue for commercial use of trained models.

Comment (LiyouZhou) [reply README.md:12]: do you think we should have separate fields for model license and data license?

File: README.md (hunk 1 above)
Comment (LiyouZhou) [reply README.md:20]: will change

File: README.md (hunk 2 above)
Comment (JWhitleyWork) [reply README.md:12]: I really do. It would help immensly for companies looking to use the models commercially.

**Comment by LiyouZhou** [comment]
@JWhitleyWork Just pushed an update, would you be able to merge this?

**Review by JWhitleyWork — APPROVED** [review]
LGTM

**Pull request merged**